src/shared_modules/dbsync/tests/sqlite/sqlite_test.cpp

```diff
@@ -237,3 +237,41 @@ TEST_F(SQLiteTest, ColumnValue)
     EXPECT_DOUBLE_EQ(4.0, spColumn5->value(double_t{}));
     EXPECT_EQ(SQLITE_FLOAT, spColumn5->type());
 }
+
+TEST_F(SQLiteTest, StatementExpand)
+{
+    std::shared_ptr<IConnection> spConnection{ new Connection };
```

Comment:
I realized the other tests do it like this, but would be better to use make_shared:
`auto spConnection = std::make_shared<IConnection>(Connection());`

Reply:
Agreed with @jftuduri

Reply:
Thanks for the review @jftuduri @sebasfalcone 

Because of a building error, and proposed by @jftuduri, I'll use `std::shared_ptr<IConnection> spConnection = std::make_shared<Connection>();` instead.